.ado/templates/react-native-init.yml

```diff
@@ -119,14 +121,14 @@ steps:
   - task: CmdLine@2
     displayName: Apply windows template (with nuget)
     inputs:
-      script: npx react-native-windows-init --verbose --version $(npmVersion) --overwrite --language ${{ parameters.language }} --projectType ${{ parameters.projectType }} ${{ parameters.additionalInitArguments }} --experimentalNuGetDependency true --nuGetTestFeed $(System.DefaultWorkingDirectory)\NuGetTestFeed
```

Comment:
I think we might want to still grab latest here, so that we grab the package that was published to Verdaccio.

Reply:
Same below